PageObjectPattern/src/test/java/serenityCucumberStepDefinitions/LoginStepDefinitions.java

```diff
@@ -0,0 +1,40 @@
+package serenityCucumberStepDefinitions;
+
+import io.cucumber.java.en.*;
+import net.thucydides.core.annotations.Steps;
+import serenity.pages.LoginPage;
+import serenity.pages.MyAccountPage;
+
+//Cucumber glue code, using regex
+public class LoginStepDefinitions {
+
+    @Steps
+    LoginPage loginPage;
+
+    @Steps
+    MyAccountPage userPage;
+
+    @Given("^Unauthenticated User is on Login Page$")
+    public void getPage() {
+        System.out.println("Step 1");
+        loginPage.openLoginPage();
+    }
+
+    @When("^User inserts (.+) , (.+)$")
```

Comment:
This step definition can be simplified to the following:
`User inserts {word} , {word}` or `User inserts {} , {}`

Other useful expressions https://cucumber.io/docs/cucumber/cucumber-expressions/

Reply:
just tried using regex)